src/wwwroot/template-editor/adapterService.js

```diff
@@ -296,13 +296,44 @@
      */
     function campaignSaveChangesAsTemplate(params) {
       traceAdapterCall(campaignSaveChangesAsTemplate, arguments);
-      var savedCampaignAsTemplate = {
-        Success: true,
-        ErrorMessage: '',
-        UrlToRedirect: 'http://www.google.com/',
-        IdTemplate: 1
-      }
-      return savedCampaignAsTemplate;
+      return saveTemplateContentChanges(params).then(function() {
```

Comment:
I think that _params_ value does not have the same structure expected by the backend.

Reply:
Never mind, you are dealing with it in saveTemplateContentChanges.

BTW, is it necessary having two methods?